go.mod

```diff
@@ -3,8 +3,8 @@ module github.com/turbot/steampipe-plugin-aws
 go 1.19
 
 require (
-	github.com/aws/aws-sdk-go v1.44.189
-	github.com/aws/aws-sdk-go-v2 v1.18.1
+	github.com/aws/aws-sdk-go v1.44.306
```

Comment:
Is this bumped to add support for the Cognito IDP endpoints?

Reply:
This update was performed when I executed `go get github.com/aws/aws-sdk-go-v2/service/cognitoidentityprovider`, I can probably revert this part.

Reply:
I've managed to downgrade `github.com/aws/aws-sdk-go` but not `github.com/aws/aws-sdk-go-v2`, `go mod tidy` will always re-upgrade it afterward.

Probably because github.com/aws/aws-sdk-go-v2/service/cognitoidentityprovider v1.23.0 is too recent:
https://github.com/aws/aws-sdk-go-v2/releases/tag/release-2023-07-13

Reply:
@pdecat I'm OK with updating the SDK versions (we should generally be on the latest one anyway), but I was asking about `github.com/aws/aws-sdk-go` specifically since we generally only use that package for endpoints.